cmd/ctr/commands/snapshot/snapshot.go

```diff
@@ -187,7 +187,7 @@ var prepareCommand = cli.Command{
 var viewCommand = cli.Command{
 	Name:      "view",
 	Usage:     "create a read-only snapshot from a committed snapshot",
-	ArgsUsage: "[flags] <key> [<parent>]",
+	ArgsUsage: "[flags] <key> <parent>",
```

Comment:
seems like these two changes are in opposition? The code fix made it possible to only specify `<key>` but the usage text change is removing the *optional* notation around `<parent>`?

Reply:
The first change is for `prepare` and the second one is for `view`.

I don't think there is a usecase to create a view of an empty parent.

Reply:
ha..sorry; wasn't looking closely enough. Thought it was the same command!

Reply:
There is a use case for empty parent on view: its just an empty file system. Not a special case and completely supported.

Reply:
what is the usecase of an empty filesystem view?

Reply:
I don't know but I'm not going to make a value judgement on it. It is part of the model and the snapshotters support it.

Let me pose the question this way: why should we special case this for a snapshot view?

Reply:
> what is the usecase of an empty filesystem view?
  
The main use case to not push it on clients to special case this condition. It is reasonable that a client may be mounting in the files needed for a container and does not care whether or not the snapshot is empty but never wants a writable layer.